lib/datadog/open_feature/extensions.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative '../core/configuration'
+require_relative 'configuration'
+
+module Datadog
+  module OpenFeature
+    module Extensions
+      def self.activate!
+        Core::Configuration::Settings.extend(Configuration::Settings)
+      end
+    end
+  end
+end
```

Comment:
I know we've used this pattern in other places in the codebase, but I suggest just inlining this where it's used -- we're basically carrying a whole new file with 14 lines of code to add this indirection.

Reply:
Thanks for the feedback